src/udev/udev-builtin-net_id.c

```diff
@@ -434,8 +491,30 @@ static int names_pci(struct udev_device *dev, struct netnames *names) {
                 if (!names->pcidev)
                         return -ENOENT;
         }
-        dev_pci_onboard(dev, names);
-        dev_pci_slot(dev, names);
+
+        vf_info.physfn_pcidev = NULL;
```

Comment:
u think it would be nicer to just initialize the whole of vf_info at time of declaration to `= {};`